include/gemmi/to_pdb.hpp

```diff
@@ -498,6 +501,7 @@ inline void write_header(const Structure& st, std::ostream& os,
           }
         }
         if (col != 0)
+          buf[80] = '\n';
           os.write(buf, 81);
```

Comment:
we'd need braces here: if (col != 0) { ... }

Reply:
Oops! Sorry & Thanks! Fixed.